src/Domain/DomainHelpers.cpp

```diff
@@ -566,32 +573,12 @@ wedge_coordinate_maps(const double inner_radius, const double outer_radius,
 
     // clang-tidy: trivially copyable
     return make_vector_coordinate_map_base<Frame::Logical, TargetFrame, 3>(
-        std::move(wedges_and_half_wedges));  // NOLINT
-  }
-
-  if (x_coord_of_shell_center != 0.0) {
-    using Identity2D = CoordinateMaps::Identity<2>;
-    using Affine = CoordinateMaps::Affine;
-    auto shift_1d = Affine{-1.0, 1.0, -1.0 + x_coord_of_shell_center,
-                           1.0 + x_coord_of_shell_center};
-    // clang-tidy: trivially copyable
-    const auto translation = CoordinateMaps::ProductOf2Maps<Affine, Identity2D>(
-        std::move(shift_1d), Identity2D{});  // NOLINT
-    return make_vector_coordinate_map_base<Frame::Logical, TargetFrame, 3>(
-        std::move(wedges), std::move(translation));  // NOLINT
+        std::move(wedges_and_half_wedges), std::move(compression), std::move(translation));  // NOLINT
```

Comment:
It looks like this change adds the functionality of having all of half wedges, compression, and translation together. Is it correct that the vector of coordinate maps will be used to compose the maps in the given order? 

If this adds a new allowed case, should there be a test that the composition of maps works as expected?

Reply:
Yup, the wedges map the logical cubes to a sphere, which is then mapped to itself (but the gridpoints are redistributed). The sphere is then translated. I'll add a test that checks that this function gives the correct coordinate maps as is already done in `test_wedge_map_generation` for the other cases.